client/conn.go

```diff
@@ -109,6 +112,18 @@ func (c *Conn) Ping() error {
 	return nil
 }
 
+// use default SSL
+// pass to options when connect
+func (c *Conn) UseSSL(insecureSkipVerify bool) {
```

Comment:
how about `insecureVerify`? while I call `UseSSL((true)`, I think it need to verify